tools/juju-bundle-applications.py

```diff
@@ -7,7 +7,7 @@
 
 for filename in sys.argv[1:]:
     with open(filename, 'r') as f:
-        data = yaml.load(f, Loader=yaml.SafeLoader)
+        data = yaml.load_all(f, Loader=yaml.SafeLoader)
         if 'applications' in data:
             application_list.update(data['applications'].keys())
 
```

Comment:
The `yaml.load_all()` function returns a list of documents:
```suggestion
        for d in data:
            if 'applications' in d:
                application_list.update(d['applications'].keys())
```